models/text_recognition_crnn/README.md

```diff
@@ -59,6 +61,31 @@ python demo.py --input /path/to/image --model text_recognition_CRNN_CN_2021nov.o
 # get help regarding various parameters
 python demo.py --help
 ```
+### C++
+
+Install latest OpenCV and CMake >= 3.24.0 to get started with:
+
+```shell
+# detect on camera input
+./build/opencv_zoo_text_recognition_crnn 
+# detect on an image
+./build/opencv_zoo_text_recognition_crnn --input /path/to/image -v
```

Comment:
need to use `-i=/path/to/image` or `--input=/path/to/image`

Reply:
using `-v` flag can't show the image

Reply:
./build/opencv_zoo_text_recognition_crnn -v=0
show result as text but won't stop without ctrl+c. Must I add a counter to stop after 100 inferences?

Reply:
I think just be as consistent as possible with other demos

Reply:
> using -v flag can't show the image

Just modify the doc here to make sure the commands work. It does not worth making 100% same behavior as Python demos.